test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenUpdateSpecs.java

```diff
@@ -570,6 +589,111 @@ public HapiApiSpec updateHappyPath() {
 				);
 	}
 
+	private HapiApiSpec onlyValidCustomFeeScheduleCanBeUpdated() {
+		final var hbarAmount = 1_234L;
+		final var htsAmount = 2_345L;
+		final var numerator = 1;
+		final var denominator = 10;
+		final var minimumToCollect = 5;
+		final var maximumToCollect = 50;
+
+		final var token = "withCustomSchedules";
+		final var feeDenom = "denom";
+		final var hbarCollector = "hbarFee";
+		final var htsCollector = "denomFee";
+		final var tokenCollector = "fractionalFee";
+		final var invalidEntityId = "1.2.786";
+
+		final var customFeesKey = "antique";
+
+		final var newHbarAmount = 17_234L;
+		final var newHtsAmount = 27_345L;
+		final var newNumerator = 17;
+		final var newDenominator = 107;
+		final var newMinimumToCollect = 57;
+		final var newMaximumToCollect = 507;
+
+		final var newFeeDenom = "newDenom";
+		final var newHbarCollector = "newHbarFee";
+		final var newHtsCollector = "newDenomFee";
+		final var newTokenCollector = "newFractionalFee";
+
+		final var newCustomFeesKey = "modern";
+
+		return defaultFailingHapiSpec("OnlyValidCustomFeeScheduleCanBeUpdated")
```

Comment:
Intentionally use `defaultFailingHapiSpec` here to fail the spec as expected. Will change to `defaultHapiSpec` after the TokenUpdateTransitionLogic supports CustomFees.